containers/session-containers/skaha-desktop/Dockerfile.centos.xfce.vnc

```diff
@@ -73,21 +73,9 @@ ADD ./src/common/scripts $STARTUPDIR
 RUN $INST_SCRIPTS/set_user_permission.sh $STARTUPDIR /headless
 
 # CANFAR Specifics Below
-
 COPY conf/xfce-applications.menu /etc/xdg/menus/xfce-applications.menu
 COPY conf/xfce-applications.menu /etc/xdg/menus/applications.menu
```

Comment:
I think these two COPY lines (and the associated files) can be removed now?  The default one in the container should work?  (Maybe I'm wrong.)

Reply:
No, the files and therefor the COPY lines are needed. They are merged with the AstroSoftware menu generated at deployment time.